client/sites/components/site-preview-pane/constants.ts

```diff
@@ -20,6 +20,7 @@ export const TOOLS_LOGS_PHP = 'tools-logs-php';
 export const TOOLS_LOGS_WEB = 'tools-logs-web';
 export const TOOLS_SFTP_SSH = 'tools-sftp-ssh';
 export const TOOLS_DATABASE = 'tools-database';
+export const ADVANCED_TOOLS = 'advanced-tools';
```

Comment:
Nit: Let's just call it `TOOLS = 'tools` for consistency!

Reply:
Renamed in 2e41f39cc15e4deaad0f6964664589e2d4ce3195